### PR TITLE
Add External/Internal and priority to compass overview

### PR DESCRIPTION
**Pull request by Williangalvani**
Screenshot 2023-02-08 at 15 40 07

### CODE REVIEW

File: core/frontend/src/components/vehiclesetup/overview/OnboardSensors.vue [hunk 1]
```diff
@@ -96,6 +98,40 @@ export default Vue.extend({
...
+                priority = '1st'
+                break
+              case 2:
+                priority = '2nd'
+                break
+              case 3:
+                priority = '3rd'
+                break
+              default:
+                priority = 'Unused'
```
Comment (patrickelectric) [inline core/frontend/src/components/vehiclesetup/overview/OnboardSensors.vue:122]: Should we use [ordinal](https://www.npmjs.com/package/ordinal) ? Os it'll not be greater than 3 ?

File: core/frontend/src/utils/deviceid_decoder.ts [hunk 2]
```diff
@@ -162,5 +163,6 @@ export default function decode(device: string, devid: number): deviceId {
     bus,
     address: toHex(address),
     devtype,
+    paramValue: devid,
```
Comment (patrickelectric) [inline core/frontend/src/utils/deviceid_decoder.ts:166]: isn't that the same as param.value ?

File: core/frontend/src/components/vehiclesetup/overview/OnboardSensors.vue (hunk 1 above)
Comment (Williangalvani) [reply core/frontend/src/components/vehiclesetup/overview/OnboardSensors.vue:122]: Ardupilot doesn't support more than 3 compasses in use

File: core/frontend/src/utils/deviceid_decoder.ts (hunk 2 above)
Comment (Williangalvani) [reply core/frontend/src/utils/deviceid_decoder.ts:166]: param here is a string  :sweat_smile:

Comment (patrickelectric) [reply core/frontend/src/utils/deviceid_decoder.ts:166]: facepalm-really

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**